river/drift/retrain.py

```diff
@@ -39,10 +41,22 @@ class DriftRetrainingClassifier(base.Wrapper, base.Classifier):
 
     """
 
-    def __init__(self, model: base.Classifier, drift_detector: base.DriftAndWarningDetector = None):
+    def __init__(
+        self,
+        model: base.Classifier,
+        drift_detector: base.DriftAndWarningDetector = None,
+        use_bkg: bool = True,
```

Comment:
We try to avoid acronyms, it can be cryptic. I would rather be obvious and call this parameter `train_in_background`